tailcfg/tailcfg.go

```diff
@@ -829,6 +831,19 @@ type Hostinfo struct {
 	//       require changes to Hostinfo.Equal.
 }
 
+// VIPService represents a service created on a tailnet from the
+// perspective of a node providing that service. These services
+// have an IP address distinct from the node's IPs.
+// Ports specify which ProtoPorts are made available by this node
+// on the service's IP.
+// Active specifies whether new requests for the service should be
+// sent to this node by control.
+type VIPService struct {
+	Ports  []ProtoPortRange
+	Name   string
```

Comment:
can you document this Name field and its usual form? and how it lines up to other things elsewhere? (apparently the IP address)

(in general, move docs from the type down to their fields, where applicable)